packages/terra-clinical-onset-picker/src/OnsetPicker.module.scss

```diff
@@ -44,4 +44,10 @@
   .year {
     min-width: var(--terra-clinical-onset-picker-year-select-min-width, 5.5em);
   }
+
+  .content-wrapper {
+    @media screen and (-ms-high-contrast: active), (-ms-high-contrast: none) { // -ms needed for icon alignment in IE
+      display: flex;
```

Comment:
Do we know what is causing the misalignment? Applying a flex display style for only IE browsers seems like it might be masking a different issue. This solution does work, but it would be ideal if we knew what was is introducing the misalignment.

Reply:
I've messed around with every box model tweak I can think of, and other than noticing that the misalignment is exactly 1rem, I can't figure out why this is initially misaligned.

Reply:
@jeremyfuksa could the issue be the default vertical-align value is baseline? I found some articles that might give more info:
- https://stackoverflow.com/questions/13548168/why-my-inline-block-divs-are-not-aligned-when-only-one-of-them-has-text
- https://developer.mozilla.org/en-US/docs/Web/CSS/vertical-align